### PR TITLE
Move Subsonic to non-free list.

### PR DESCRIPTION
**Pull request by alexfornuto**
Now that Subsonic is [no longer open-source](http://forum.subsonic.org/forum/viewtopic.php?f=4&t=16604#p71128), it belongs on the non-free list.

### CODE REVIEW

**Comment by Kickball** [comment]
Please remove the full stop at the end of the item and add an extra closing bracket in its place.

**Comment by alexfornuto** [comment]
@Kickball Which file are you referring to, and what exactly do you mean? In one file I simply removed a line, in another I created one exactly formatted as the one above it.

**Comment by chid** [comment]
I would recommend replacing Subsonic's link with https://github.com/Libresonic/libresonic

```Libresonic is maintained by Eugene E. Kashpureff Jr. It originated as an unofficial("Kang") of Subsonic which did not contain the Licensing code checks present in the official builds. With the announcement of Subsonic's closed-source future, a decision was made to make a full fork and rebrand to Libresonic.```

**Comment by nodiscc** [comment]
@chid Can you send a patch to add libresonic (in another Pull Request)? I think it would be accepted. The only downside is that there's no git history of the development (https://github.com/Libresonic/libresonic/commits/develop - everything was imported in a single fat commit)

**Comment by alexfornuto** [comment]
> The only downside is that there's no git history of the development (https://github.com/Libresonic/libresonic/commits/develop - everything was imported in a single fat commit)

That's because the original import of Subsonic to GitHub was in a single commit, so it has(had) the same issue. 

> I would recommend replacing Subsonic's link with https://github.com/Libresonic/libresonic

That fork had not been created when I made this PR (as far as I recall). @nodiscc I can update this PR, there's no need for a new one.

**Comment by nodiscc** [comment]
> I can update this PR, there's no need for a new one. 

Yep It would be nice; otherwise this PR is ready to merge.

**Comment by alexfornuto** [comment]
Will do, might take a day or two, but until then this should probably keep the `needs work` tag.

File: README.md [hunk 1]
```diff
@@ -587,8 +587,7 @@ See also <https://en.wikipedia.org/wiki/List_of_streaming_media_systems>, <https
   * [Mopidy](http://mopidy.readthedocs.org/) - Extensible music server. Offers a superset of the mpd API, as well as integration with 3rd party services like Spotify, SoundCloud etc. ([Source Code](https://github.com/mopidy/mopidy)) `Apache` `Python`
   * [mpd](http://www.musicpd.org/) - Daemon to remotely play music, stream music, handle and organize playlists. Many clients available. `GPLv2` `C++`
   * [Sonerezh](https://www.sonerezh.bzh/) - A self-hosted, web-based application for stream your music, everywhere. ([Demo](https://www.sonerezh.bzh/demo/), [Documentation](https://www.sonerezh.bzh/docs/), [Source Code](https://github.com/sonerezh/sonerezh)) `GPLv3` `PHP`
-  * [Subsonic](http://subsonic.org/) - Open-source web-based media streamer and jukebox. ([Demo](http://demo.subsonic.org/login.view?user=guest4&password=guest), [Source Code](https://github.com/EugeneKay/subsonic)) `GPLv3` `Java`
-
+  * [Libresonic](http://libresonic.org/) - Open-source web-based media streamer and jukebox. A fork of Subsonic's last open-source release, before it switched licenses. `GPLv3` `Java`
```
Comment (alexfornuto) [inline README.md]: No source link since the URL goes directly to the GitHub repo.

**Comment by alexfornuto** [comment]
Aight, should be good to go now.

**Comment by nodiscc** [comment]
@alexfornuto please reorder the section alphabetically.

**Comment by alexfornuto** [comment]
image

**Pull request merged**